README.md

```diff
@@ -129,6 +130,45 @@ Run SortingHat frontend Vue app:
 $ yarn serve
 ```
 
+
+## SortingHat service
+
+Starting at version 0.8, SortingHat is released with a server app. The server has two
+modes, `production` and `development`.
+
+When `production` mode is active, a WSGI app is served. The idea is to use a reverse
+proxy like NGINX or similar, that will be connected with the WSGI app to provide
+an interface HTTP.
+
+When `development` mode is active, an HTTP server is launched, so you can interact
+directly with SortingHat using HTTP requests. Take into account this mode is not
+suitable nor safe for production.
+
+You will need a django configuration file to run the service. You can use and modify
+one of the ones stored in `config/settings` folder. The file must be accessible
+via `PYTHONPATH` env variable. For the next examples, we will use `devel.py` file.
+
+In order to run the service for the first time, you need to execute the next commands:
+
+Build the UI interface:
+```
+$ yarn build
```

Comment:
this should include `cd ui` to know from where to run the yarn command

Reply:
Fixed.